scripts/testnet/prerequisites.sh

```diff
@@ -9,17 +9,32 @@ export DISTRIBUTION=$(cat /etc/os-release | grep "^ID=" | sed 's/ID=//')
 if [[ "$DISTRIBUTION" =~ ^(fedora|centos|rhel)$ ]]; then
   export PACKAGE_MANAGER="dnf"
   export REQUIRED_PACKAGES="git golang gcc lsof jq curl"
-  echo "Using DNF to install required packages: $REQUIRED_PACKAGES"
-fi
+  export INSTALL_PACKAGES_COMMAND="sudo $PACKAGE_MANAGER install -y $REQUIRED_PACKAGES"
 
-if [[ "$DISTRIBUTION" =~ ^(ubuntu|debian)$ ]]; then
+  echo "Using DNF to install required packages: $REQUIRED_PACKAGES"
+elif [[ "$DISTRIBUTION" =~ ^(ubuntu|debian)$ ]]; then
   export PACKAGE_MANAGER="apt-get"
   export REQUIRED_PACKAGES="git gcc lsof jq curl"
+  export INSTALL_PACKAGES_COMMAND="sudo $PACKAGE_MANAGER install -y $REQUIRED_PACKAGES"
 
   echo "Using APT to install required packages: $REQUIRED_PACKAGES"
+elif [[ "$DISTRIBUTION" =~ ^(arch)$ ]]; then
+  export PACKAGE_MANAGER="pacman"
+  export REQUIRED_PACKAGES="git gcc lsof jq curl"
+  export INSTALL_PACKAGES_COMMAND="sudo $PACKAGE_MANAGER -S $REQUIRED_PACKAGES"
+
+  echo "Using PACMAN to install required packages: $REQUIRED_PACKAGES"
 fi
 
-sudo $PACKAGE_MANAGER install -y $REQUIRED_PACKAGES
+if [ -z "$INSTALL_PACKAGES_COMMAND" ]; then
+  echo "Your operating system was not identified. The required packages were not installed, however they could still be part of your system."
+  read -r -p "Would you like to try to continue anyway? [Y/n] " response
+  if [[ "$response" =~ ^[Nn][Oo]?$ ]]; then
+    exit
+  fi
+else
+  $INSTALL_PACKAGES_COMMAND
```

Comment:
Add `echo "Using $PACKAGE_MANAGER to install required packages: $REQUIRED_PACKAGES"` before this line, then you can delete all `echo`s  above.

Reply:
Good suggestion!